test/integration/query-interface/schemas.test.ts

```diff
@@ -0,0 +1,20 @@
+import { expect } from 'chai';
+import { sequelize, getTestDialect } from '../support';
+
+const queryInterface = sequelize.queryInterface;
+
+const testSchema = 'testSchema';
+const dialect = getTestDialect();
```

Comment:
we call this `dialectName` by convention. `dialect` is always the dialect instance (`sequelize.dialect`)